--- rgeos.orig/src/Makevars.win	2019-04-23 08:39:33.000000000 -0300
+++ rgeos/src/Makevars.win	2019-06-27 15:57:54.180025213 -0300
@@ -1,14 +1,11 @@
 VERSION = 2.2.3
 COMPILED_BY ?= gcc-4.6.3
-RWINLIB = ../windows/gdal2-$(VERSION)
 TARGET = lib$(subst gcc,,$(COMPILED_BY))$(R_ARCH)
 
-PKG_CPPFLAGS =\
-	-I$(RWINLIB)/include/geos
+PKG_CPPFLAGS = -I$(LIBRARY_INC)
 
 PKG_LIBS = \
-	-L$(RWINLIB)/$(TARGET) \
-        -L$(RWINLIB)/lib$(R_ARCH) \
+    -L$(LIBRARY_LIB) \
 	-lgeos_c -lgeos
 
 all: clean winlibs
