Index: configure.ac
===================================================================
--- configure.ac	(revision 689)
+++ configure.ac	(working copy)
@@ -24,15 +24,8 @@
 
 AC_MSG_NOTICE([${PACKAGE_NAME}: ${PACKAGE_VERSION}])
 
-AC_CHECK_FILE([/usr/bin/svnversion],
- [SVN_VERSION=`svnversion -n '.'`],
- [SVN_VERSION=""])
-
-if test -n "${SVN_VERSION}" && test "${SVN_VERSION}" != "exported" && test "${SVN_VERSION}" != "Unversioned directory"; then
-  echo "${SVN_VERSION}" > inst/SVN_VERSION
-else
-  SVN_VERSION=`cat inst/SVN_VERSION | tr -d '\n'`
-fi
+# CRAN tarball so, assume `inst/SVN_VERSION` is accurate
+SVN_VERSION=`cat inst/SVN_VERSION | tr -d '\n'`
 AC_MSG_NOTICE([svn revision: ${SVN_VERSION}])
 
 GEOS_CONFIG="geos-config"
